src/main/scala/firrtl/passes/ReplSeqMem.scala

```diff
@@ -0,0 +1,142 @@
+package firrtl.passes
+
+import com.typesafe.scalalogging.LazyLogging
+import firrtl._
+import firrtl.ir._
+import Annotations._
+import java.io.Writer
+import AnalysisUtils._
+
+sealed trait PassOption
+case object InputConfigFileName extends PassOption
+case object OutputConfigFileName extends PassOption
+case object PassCircuitName extends PassOption
+
+object Error {
+  def apply[T <: Any](msg: String) = throw new Exception(msg)
```

Comment:
I just wanted a shorthand for not having to type throw new Exception(msg) everytime. I'm ok with whatever/will look into that.